cocotb/handle.py

```diff
@@ -161,7 +160,7 @@ class RegionObject(SimHandleBase):
     """
     def __init__(self, handle, path):
         SimHandleBase.__init__(self, handle, path)
-        self._discovered = False
+        self._discovered : bool = False  #: ``True`` if this object has already been discovered.
```

Comment:
This syntax isn't legal in python 3.5, hence the CI failure.

Reply:
Fixed.